app/models/tenant.rb

```diff
@@ -184,7 +184,7 @@ def project?
   end
 
   def visible_domains
-    MiqAeDomain.where(:tenant_id => ancestor_ids.append(id)).joins(:tenant).order('tenants.ancestry DESC NULLS LAST, priority DESC')
+    MiqAeDomain.where(:tenant_id => ancestor_ids + [id]).joins(:tenant).order('tenants.ancestry DESC NULLS LAST, priority DESC')
```

Comment:
Think `path_ids` is what we want.

Reply:
Didn't know it was a thing.  Will change.